final/recommendations/backend.tex

```diff
@@ -30,6 +30,22 @@ \subsection{More extensive history functionality}
 
 \subsection{Analysis in context}
 
+In \cref{sec:dynsem-refactor} it was explained how DynSem was refactored to
+allow passing in an existing context in which REPL commands should be
+evaluated. Besides evaluating a complete or partial program, the REPL also
+offers the user a command to analyze a program without evaluating it.
```

Comment:
Analysis in context should also work when evaluating an expression, shouldn't it?